.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!-- This comments are hidden when you submit the issue, so you do not need to remove them! -->
+<!-- Please be sure to check out our [contributing guidelines](https://github.com/sunpy/sunpy/blob/master/CONTRIBUTING.rst). -->
+<!-- Please be sure to check out our [code of conduct](https://github.com/sunpy/sunpy/blob/master/CODE_OF_CONDUCT.rst). -->
```

Comment:
Will these links be rendered in any way if that line is commented? They won't be when "preview" and when "editing" will make harder to copy - the last parenthesis will be also selected by default.